src/net/websocket.cc

```diff
@@ -11,11 +11,13 @@
 enum PROTOCOL_ID {
   WS_HTTP = 0,
   WS_ASCII = PROTOCOL_WS_ASCII,
+  WS_JSON = PROTOCOL_WS_JSON,
 };
 
 static struct lws_protocols protocols[] = {
     {"http", lws_callback_http_dummy, 0, 0, WS_HTTP},
     {"ascii", ws_ascii_callback, sizeof(struct ws_ascii_session), 4096, WS_ASCII, NULL, 0},
+    {"json", ws_ascii_callback, sizeof(struct ws_ascii_session), 4096, WS_JSON, NULL, 0},
```

Comment:
the callbacks are wrong , also the sesion struct name

Reply:
Thanks for the review. I backed out the local_options change, put back the DEBUG_CHECK in ws_ascii.cc including both ascii/json. I'm just trying to add an alternate name to the existing ascii websocket protocol (json) that simply results in adding {"message": ...} around what it already sends to the client but use the exact same ws_asscii_callback/session code. Should I really duplicate this callback/structure code with another name? Thanks for the help!